### ISSUE TITLE
Error 500 when trying to login user in production (docker setup)

### ISSUE THREAD

**Issue by jduraj** [pydanny/cookiecutter-django#1105]
Hi,

Thanks a lot for this nice project.

When I try to login using my superuser credentials on production user docker setup, it fails with 500 error..

And then, when I check the docker logs, I don't really see anything except lines like:
DEBUG 2017-04-03 20:13:56,294 base 15 140604741359360 Configuring Raven for host: <raven.conf.remote.RemoteConfig object at 0x7fe10da72f28>
INFO 2017-04-03 20:13:56,297 base 15 140604741359360 Raven is not configured (logging is disabled). Please see the documentation for more information.

Is it not possible to see logs the usual way in production?

Sorry if noob questions..
Thanks

**Comment by pydanny** [comment]
Marking this as closed, please reopen if still an issue.

**Comment by jduraj** [comment]
For people that might be stuck at the same point, I managed to resolve this issue by using sentry.
I first created a Sentry account, and then provided the SENTRY_DSN in the .env file.

This allowed me to obtain a useful trace in the Sentry App.
Basically, the 500 error was due to a failure in the Mail backend, as I didn't provide a Mail backend configuration in the .env file (e.g. Mailgun), so I proceeded to add the Mailgun API and the Mailgun domain in the .env file and this solved my issue.
